src/saving/ISaveUpgradeStep.cs

```diff
@@ -161,6 +161,19 @@ private void UpgradeBehaviouralValues(JProperty property, JEnumerable<JProperty>
         }
     }
 
+    internal class UpgradeStep0561To057 : BaseRecursiveJSONWalkerStep
+    {
+        protected override string VersionAfter => "0.5.7.0-rc1";
+
+        protected override void CheckAndUpdateProperty(JProperty property)
+        {
+            if (property.Name.Contains("GameWorld"))
```

Comment:
Do we need to also upgrade patches or just the GameWorld?

Reply:
Just the GameWorld apparently, the patches seem to have `eventsLog` initialized already on save upgrade, as evidenced by upgrading and then loading an older save which will show a "No events recorded" text under passed time periods on the local timeline.

Relevant code line: https://github.com/Revolutionary-Games/Thrive/blob/4eb6e8fd5704d1098d8ba8317ea6fc8a9b510763/src/microbe_stage/editor/TimelineTab.cs#L233

Reply:
I guess it's fine then if it is intended that the data might be missing.